opencti-platform/opencti-front/src/private/components/data/csvMapper/representations/attributes/CsvMapperRepresentationAttributeForm.tsx

```diff
@@ -53,7 +53,7 @@ CsvMapperRepresentationAttributeFormProps
   const { name, value } = field;
   const { setFieldValue } = form;
 
-  const options = alphabet(1);
+  const options = alphabet(26);
```

Comment:
it's a bit unclear but alphabet(1) generates column names Afrom  to ZZ
alphabet(26) would generate names from A to ZZZZZ...ZZZZ (27 'Z'). That's a bit too much :)

Reply:
Thats not what i have experimented. 
alphabet(1) -> A to AZ
alphabet(26) -> A to ZZ

Reply:
My bad!